### ISSUE TITLE
why decoder produce same generated summary ? 

### ISSUE THREAD

**Issue by PH-github95** [theamrzaki/text_summurization_abstractive_methods#20]
The model 4 had a loss of 5 and there have always the same generated summary (for all articles ):

[UNK] [UNK] , 28 , has been charged with two counts of first-degree murder . he has been charged with two counts of attempted murder . he was sentenced to 15 years in prison and sentenced to 18 months in prison .

**Comment by theHimanshuShekhar** [comment]
@PH-github95 did you solve this issue?
can you tell me how you did it?
I am facing this issue as well. Any help would be greatly appreciated

**Comment by DengYangyong** [comment]
I meet the sample problem as you when I train the model in chinese dataset.
In addtion, the loss is too high which was 11 at the beggining and decreased to about 6 after 30K steps.
There may be same problem with the code as when I use the pytorch version ( https://github.com/rohithreddy024/Text-Summarizer-Pytorch) to train the model , the loss ranged from 6 to 0.8 and the results were good.
